gradle-jdks/src/main/java/com/palantir/gradle/jdks/JdksExtension.java

```diff
@@ -74,6 +77,14 @@ public final void setDaemonTarget(int value) {
         getDaemonTarget().set(JavaLanguageVersion.of(value));
     }
 
+    public final Property<Boolean> getDaemonJdkOnly() {
+        return daemonJdkOnly;
+    }
+
+    public final void setDaemonJdkOnly(Boolean value) {
+        getDaemonJdkOnly().set(value);
+    }
```

Comment:
I'd actually hide this property maybe so it doesn't become a breaking change if we do further baseline-java-versions integration we talked about. ie no readme for it. No setter, we can make a method called `daemonJdkOnly()` that sets it to true. Then later we can make this method do `jdkMajorVersionsToUse.set(daemonTargert.map(Set::of))` later and no breaks.

Reply:
Actually, we could have this `jdkMajorVersionsToUse` property right now, it moves the logic from `JdkConfigurationDistributor`. Later we just give it a different default value (now it just needs to get all the keys from `jdks` as per in `JdkConfigurationDistributor`).

Reply:
I think `jdkMajorVersionsToUse` is a generally useful thing that could be used by external people (probably even worth documenting that and `daemonJdkOnly`).